server/service/orbit_client.go

```diff
@@ -78,3 +79,15 @@ func (oc *OrbitClient) GetConfig(orbitNodeKey string) (json.RawMessage, error) {
 
 	return resp.Flags, nil
 }
+
+func (oc *OrbitClient) Ping() error {
+	verb, path := "HEAD", "/api/latest/fleet/orbit_ping"
```

Comment:
this path changes in a follow-up PR: https://github.com/fleetdm/fleet/pull/7933/files

Reply:
But maybe we'd like to merge this PR without merging #7933?

(Looks like CI fails because of this path.)

Reply:
the e2e test is consistently failing in `main` too, looking at what the test does, I'm almost sure it's unrelated to the changes in this PR.

Reply:
that is weird and looks like it's an error/status code returned by clouflared. Look at the "Start Tunnel" step, seems like it silently failed:

```
 2022-09-26T11:53:36Z WRN Failed to determine the IPv6 for this machine. It will use 2001:db8:: as source IP for error messages such as ICMP TTL exceed error="dial udp [2606:4700:4700::1111]:53: connect: network is unreachable" connIndex=3 ip=198.41.192.77
2022-09-26T11:53:36Z INF Connection ab686873-162c-4c2b-9f63-3b8969846644 registered connIndex=3 ip=198.41.192.77 location=DFW
```

 I triggered a re-run, will keep an eye on it

Reply:
actually, I'm not sure what I said is 100% correct

Reply:
what I'm pretty sure: it's not related to the path change, as the test passed in that PR: https://github.com/fleetdm/fleet/actions/runs/3127482305

Reply:
Ya, the HTTP 530 does sound like a `cloudflared` thing.

Reply:
I too ran into `HTTP 530` last week, and the next re-run fixed it for me